worker/codalabworker/worker.py

```diff
@@ -175,26 +147,30 @@ def _get_installed_memory_bytes(self):
             # Fallback to sysctl when os.sysconf('SC_PHYS_PAGES') fails on OS X
             return int(check_output(['sysctl', '-n', 'hw.memsize']).strip())
 
-    def _get_allocated_memory_bytes(self):
-        return sum(self._worker_state_manager.map_runs(lambda run: run.requested_memory_bytes))
-
     def _get_memory_bytes(self):
-        return max(0, self._get_installed_memory_bytes() - self._get_allocated_memory_bytes())
+        return self._get_installed_memory_bytes()
```

Comment:
we dont report the leftover cpuset and gpusets of a worker; but we report the leftover memory; this change is made so that the worker reports its total memory, like cpuset and gpusets